### ISSUE TITLE
[BUG] Pyarmor does not install via Pip on M2 Macbook Air OS 13.3.1

### ISSUE THREAD

**Issue by KyleMylonakisProtopia** [dashingsoft/pyarmor#1127]
1. A clear and concise description of the question (required)
Installing pyarmor on a M2 macbook air fails as follows.

```shell
pip install pyarmor==8.1.0

Collecting pyarmor==8.1.0
  Downloading pyarmor-8.1.0-py2.py3-none-any.whl (2.3 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 2.3/2.3 MB 40.7 MB/s eta 0:00:00
ERROR: Could not find a version that satisfies the requirement pyarmor.cli.core~=2.1.0 (from pyarmor) (from versions: none)
ERROR: No matching distribution found for pyarmor.cli.core~=2.1.0
```

It seems suspicious since pip does not seem to be downloading the MacOS version of the distribution.

I was hoping the newly released Apple Silicon would support this, but there still seem to be issues. I cleared my python and anaconda caches and created new environments, but it still seems like an incorrect version is being downloaded.

**Comment by Kaiyangshi-Ito** [comment]
I tried both `pip install pyarmor==8.1.1 `  and `pip install pyarmor==8.1.0`, got 
```
ERROR: Could not find a version that satisfies the requirement pyarmor.cli.core~=2.1.1 (from pyarmor) (from versions: none)
ERROR: No matching distribution found for pyarmor.cli.core~=2.1.1
```
and 
```
ERROR: Could not find a version that satisfies the requirement pyarmor.cli.core~=2.1.0 (from pyarmor) (from versions: none)
ERROR: No matching distribution found for pyarmor.cli.core~=2.1.0
```

**Comment by jondy** [comment]
I'll check this, as a quick workaround, download `pyarmor.cli.core` wheel file from PyPI directly, install from file directly

**Comment by jondy** [comment]
@Kaiyangshi-Ito 

Would you test your machine support this tag?

```
python -m pip debug -v | grep none-macosx_11_0_arm64
```

**Comment by Kaiyangshi-Ito** [comment]
> 

I typed `python -m pip debug -v | grep none-macosx_11_0_arm64`, got: 
```
WARNING: This command is only meant for debugging. Do not use this with automation for parsing and getting these details, since the output and options of this command may change without notice.
  cp39-none-macosx_11_0_arm64
  py39-none-macosx_11_0_arm64
  py3-none-macosx_11_0_arm64
  py38-none-macosx_11_0_arm64
  py37-none-macosx_11_0_arm64
  py36-none-macosx_11_0_arm64
  py35-none-macosx_11_0_arm64
  py34-none-macosx_11_0_arm64
  py33-none-macosx_11_0_arm64
  py32-none-macosx_11_0_arm64
  py31-none-macosx_11_0_arm64
  py30-none-macosx_11_0_arm64
```

**Comment by jondy** [comment]
Thanks, this issue will be fixed in next release.

**Comment by jondy** [comment]
Fixed it in `8.1.3`

**Comment by Kaiyangshi-Ito** [comment]
I tried `pyarmor reg pyarmor-regfile-******.zip` with the latest version, _upgraded successfully by `pip install pyarmor==8.1.3`_, got:
```
INFO     Python 3.9.13
INFO     Pyarmor 8.1.1 (trial), 000000, non-profits
INFO     Platform darwin.aarch64
INFO     register "/Users/my_username/Documents/pyarmor reg files/pyarmor-regfile-******.zip"
INFO     update license token
CRITICAL unknown error, please check pyarmor.error.log
ERROR    dlopen(/Users/my_username/anaconda3/lib/python3.9/site-packages/pyarmor/cli/core/pytransform3.so, 0x0002): Library not loaded: '@rpath/Frameworks/Python.framework/Versions/3.9/Python'
  Referenced from: '/Users/my_username/anaconda3/lib/python3.9/site-packages/pyarmor/cli/core/pytransform3.so'
  Reason: tried: '/Users/my_username/anaconda3/bin/../Frameworks/Python.framework/Versions/3.9/Python' (no such file), '/System/Library/Frameworks/Python.framework/Versions/3.9/Python' (no such file), '/Library/Frameworks/Python.framework/Versions/3.9/Python' (no such file), '/Users/my_username/anaconda3/bin/../Frameworks/Python.framework/Versions/3.9/Python' (no such file), '/System/Library/Frameworks/Python.framework/Versions/3.9/Python' (no such file), '/Library/Frameworks/Python.framework/Versions/3.9/Python' (no such file), '/Users/my_username/anaconda3/bin/../lib/Frameworks/Python.framework/Versions/3.9/Python' (no such file), '/Users/my_username/anaconda3/bin/../lib/Frameworks/Python.framework/Versions/3.9/Python' (no such file), '/Library/Frameworks/Python.framework/Versions/3.9/Python' (no such file), '/System/Library/Frameworks/Python.framework/Versions/3.9/Python' (no such file)
```
And it turns out that if I type `pyarmor -v`, it gives me:
```
Pyarmor 8.1.1 (trial), 000000, non-profits

License Type    : pyarmor-trial
License No.     : pyarmor-vax-000000
License To      : 

BCC Mode        : No
RFT Mode        : No

Notes
* Internet connection is required to verify Pyarmor license
* Trial license can't obfuscate big script and mix str
```
